### ISSUE TITLE
Migrate desisim to use specsim instead of quicksim

### ISSUE THREAD

**Issue by dkirkby** [desihub/desisim#44]
This will involve changes to some code and the desisim dependencies.

**Comment by dkirkby** [comment]
`quickbrick` is already using specsim (after #50) but quickgen is still using `desimodel.simulate`.

**Comment by rstaten** [comment]
Is specsim supposed to have the exact same output as quicksim in terms of counts? I made changes where appropriate last night, but wanted to make sure everything was done correctly before merging the new version of quickgen.

**Comment by dkirkby** [comment]
I don't expect the results to be numerically identical (since there have been some minor updates), but any differences *should* be insignificant.

**Comment by rstaten** [comment]
I did not generate another pull request after taking out the unnecessary import and changing lines where the import was implemented. Should I go ahead and do this?

**Comment by dkirkby** [comment]
PRs remain "live" after you create them, with new commits automatically triggering the test suite on TravisCI (but I don't see any test report on the PR, so I guess that isn't configured yet).  I missed your earlier update on this PR, but normally that would be enough.
